Document/0x06f-Testing-Local-Authentication.md

```diff
@@ -198,6 +198,8 @@ It is important to remember that Local Authentication framework is an event-base
 - Verify that sensitive processes, such as re-authenticating a user triggering a payment transaction, are protected using the Keychain services method.
 - Verify that the `kSecAccessControlTouchIDAny` or `kSecAccessControlTouchIDCurrentSet` flags are set and `kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly` protection classes are set when the `SecAccessControlCreateWithFlags` method is called. Note that, alternatively, `kSecAccessControlUserPresence` can be used as a flag as well when you want to be able to use passcode as a fallback. Last, note that, when `kSecAccessControlTouchIDCurrentSet` is set, changing the fingerprints registered to the device will invalidate the entry which is protected with the flag.
```

Comment:
Some remarks:
- I can't find an explanation on why we should test for kSecAccessControlTouchIDAny (or the other flags) being set
- kSecAccessControlTouchIDAny is since iOS 11.3 deprecated, use kSecAccessControlBiometryAny instead.

Reply:
Thanks! Updated it :).